specification/azureactivedirectory/resource-manager/Microsoft.Aadiam/stable/2017-04-01/examples/AzureADPrivateLinkResourceListGet.json

```diff
@@ -0,0 +1,29 @@
+
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "rg1",
+    "policyName": "ddb1",
+    "api-version": "2015-04-08"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/Default/providers/Microsoft.DocumentDb/databaseAccounts/ddb1/privateLinkResources/sql",
```

Comment:
>Microsoft.DocumentDb/databaseAccounts [](start = 103, length = 37)

`type` doesn't match `id` in this example. Copy/paste issue?

Reply:
Fixed it.